src/test/datascience/widgets/commUtils.ts

```diff
@@ -22,8 +22,10 @@ export function initializeWidgetComms(serviceContainer: IServiceContainer): Util
     const disposable = messageChannel.onDidReceiveMessage(async ({ editor, message }) => {
         traceInfo(`Received message from Widget renderer ${JSON.stringify(message)}`);
```

Comment:
Any message sent to extension host gets logged here.
Hence no need to explicitly handle the `log` command for logging purposes.